src/Serilog.Sinks.Loggly/Sinks/Loggly/ExceptionDetails.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Serilog.Sinks.Loggly
+{
+    class ExceptionDetails
+    {
+        public string Type { get; set; }
```

Comment:
Since these are immutable and only set at construction time, could we use `{ get; }`-only properties, and initialize them via the constructor?